### PR TITLE
Jd 10 in class challenge2

### PR DESCRIPTION
**Pull request by dannysanchez559**
## Changes
_List Changes Introduced by this PR_
1. Add working fetch call and populate UI

### CODE REVIEW

File: movie-app/src/App.js [hunk 1]
```diff
@@ -1,29 +1,48 @@
...
+
+    this.state = {
+      movieTest: {},
+    }
+  }
+
+  async componentDidMount () {
+    try {
+      const movie = await getMoviesByName('Batman');
+      console.log(movie);
```
Comment (kmausolf) [inline movie-app/src/App.js:17]: Log here.

**Review by BlackCubes — DISMISSED** [review]
Whew!! One little thing on returning an async function, but you did good!

File: movie-app/src/components/MovieCard.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+
+
+const MovieCard = (props) => {
+
+    return (
+        <div className='moviecard'>
+            <img 
+                src={props.posterUrl} 
+                alt={`${props.title} poster`}
```
Comment (kmausolf) [inline movie-app/src/components/MovieCard.js:10]: Great attention to accessibility features!

**Review by JayDampitan — APPROVED** [review]
Great job

**Review by BlackCubes — APPROVED** [review]
Noice!

**Pull request merged**